src/plugin.ts

```diff
@@ -178,7 +178,7 @@ export class Plugin implements IPlugin {
       return cmd
     }
     const cmd = fetch()
-    if (!cmd && opts.must) error(`command ${id} not found`)
+    if (!cmd && opts.must) error(`command ${id} file found, but without an implementation`)
```

Comment:
it is possible that the file does not exist as well. I think we should check both cases